PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3400,6 +3400,8 @@ bool isUrl(TCHAR * text, int textLen, int start, int* segmentLen)
 
 void Notepad_plus::addHotSpot(ScintillaEditView* view)
 {
+	if (_isAttemptingCloseOnQuit)
+		return; // don't recalculate URLs when shutting down

```

Comment:
It's good to keep! 👍

Reply:
I recommend you use [this PR](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14931) instead and close this one, because that PR better reflects the fact that it doesn't relate to the originally linked issue #13916, and it's just a quality-of-life improvement.